### ISSUE TITLE
Tokens pagination is not working

### ISSUE THREAD

**Issue by 0xtzData** [blockscout/blockscout#7559]
https://eth-goerli.blockscout.com/tokens
When selecting pages 1-2-3 we are receiving next page params:
```
   "next_page_params":{
      "holder_count":18067,
      "items_count":50,
      "market_cap":"486926096.51626855",
      "name":"Frax Share"
   }
```

```
   "next_page_params":{
      "holder_count":null,
      "items_count":100,
      "market_cap":"183513290.9587705",
      "name":"0x Protocol Token"
   }
```

```
   "next_page_params":{
      "holder_count":18067,
      "items_count":150,
      "market_cap":"486926096.51626855",
      "name":"Frax Share"
   }
```
Pagination is not working without filters and with ERC-20 filter turned on.
It works with other filter groups.

**Comment by 0xtzData** [comment]
From https://github.com/blockscout/blockscout/tree/v5.1.5-beta version pagination stopped working with/without all the filters.

**Comment by 0xtzData** [comment]
https://eth-goerli.blockscout.com/tokens

**Comment by vbaranov** [comment]
FIxed in the latest front release 1.4.0
